wallet/chain/p/builder/builder.go

```diff
@@ -935,15 +1023,22 @@ func (b *builder) getBalance(
 //     place into the staked outputs. First locked UTXOs are attempted to be
 //     used for these funds, and then unlocked UTXOs will be attempted to be
 //     used. There is no preferential ordering on the unlock times.
+//   - [excessAVAX] contains the amount of extra AVAX that spend can produce in
+//     the change outputs in addition to the consumed and not burned AVAX.
+//   - [ownerOverride] optionally specifies the output owners to use for the
+//     unlocked AVAX change output if no additional AVAX was needed to be
+//     burned. If this value is nil, the default change owner is used.
 func (b *builder) spend(
 	amountsToBurn map[ids.ID]uint64,
 	amountsToStake map[ids.ID]uint64,
+	excessAVAX uint64,
+	ownerOverride *secp256k1fx.OutputOwners,
 	options *common.Options,
 ) (
-	inputs []*avax.TransferableInput,
-	changeOutputs []*avax.TransferableOutput,
-	stakeOutputs []*avax.TransferableOutput,
-	err error,
+	[]*avax.TransferableInput,
+	[]*avax.TransferableOutput,
+	[]*avax.TransferableOutput,
```

Comment:
I think it's worth still differentiating between the two outputs as part of the signature. Input and error can just be `_`.

Reply:
Just removed the diff entirely.